src/v/security/acl.h

```diff
@@ -499,4 +499,8 @@ class acl_binding_filter
     acl_entry_filter _acl;
 };
 
+/// Name of the principal the kafka client for auditing will be using
+static const acl_principal audit_principal{
```

Comment:
This header is included from a lot of compilation units, `inline` should be preferred over `static`
```suggestion
inline const acl_principal audit_principal{
```